123 rename contacts to idprovider (#135)
direct-state-transfer/dst-go: glossary.md

| Participant Address | Address used for representing a user in a channel. Currently, the off-chain address is reused as the participant address.|
|Comm Address|Address for establishing physical network connection for off-chain communication.|
|Comm Type|Type of physical network connection protocol for off-chain communication.|

> |Alias| A user-friendly used for referring to a peer in user API calls. It is assigned by the user in his/her idprovider file. |

Reviewer: ```suggestion
|Alias| A user-friendly used for referring to a peer in user API calls. It is assigned by the user in his/her ID provider file. |
```

> |Alias| A user-friendly used for referring to a peer in user API calls. It is assigned by the user in his/her ID provider file. |

Reviewer: ```suggestion
|Alias| A user-friendly name to reference a peer ID in user API calls. It is assigned by the user when registering the peer ID in his/her ID provider. |
```

The change looks good with respect to this issue. Suggestion is only a general improvement.

Author: [Done](https://github.com/hyperledger-labs/perun-node/pull/135/commits/c55f1f275ed66aae884d2380810cc6c5c06190e1)
